me.amankhanna.opendeck.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>me.amankhanna.opendeck</id>
+
+  <name>OpenDeck</name>
+  <summary>A cross-platform desktop application for stream controller devices</summary>
+
+  <developer id="me.amankhanna">
+    <name>Aman Khanna</name>
+  </developer>
+
+  <url type="vcs-browser">https://github.com/ninjadev64/OpenDeck</url>
+  <url type="homepage">https://github.com/ninjadev64/OpenDeck</url>
+  <url type="bugtracker">https://github.com/ninjadev64/OpenDeck/issues</url>
+  <url type="faq">https://github.com/ninjadev64/OpenDeck?tab=readme-ov-file#support</url>
+  <url type="contribute">https://github.com/ninjadev64/OpenDeck?tab=readme-ov-file#building-from-source--contributing</url>
+  <url type="donation">https://github.com/sponsors/ninjadev64</url>
+  <url type="contact">https://matrix.to/#/#opendeck:amankhanna.me</url>
+
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <content_rating type="oars-1.1" />
+
+  <description>
+    <p>OpenDeck is a desktop application for using stream controller devices like the Elgato Stream Deck. OpenDeck conforms to the OpenAction API, which is cross-compatible with the Stream Deck SDK, allowing a wide range of pre-existing plugins to be used.</p>
+    <p>OpenDeck supports ProntoKey, Elgato Stream Deck and some Ajazz hardware on all three major desktop platforms. If you would like to contribute support for additional hardware (e.g. Loupedeck) feel free to reach out on any of the support forums and make a pull request!</p>
+    <p>Please note that you may need to install udev subsystem rules for your device to be detected. For more information, see the Installation section of the project README on GitHub.</p>
+  </description>
+
+  <launchable type="desktop-id">me.amankhanna.opendeck.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/ninjadev64/OpenDeck/raw/main/.github/readme/mainmenu.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/ninjadev64/OpenDeck/raw/main/.github/readme/multiaction.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/ninjadev64/OpenDeck/raw/main/.github/readme/plugins.png</image>
```

Comment:
these screenshots should be from a commit, not `main`

Reply:
Done